Makefile.am

```diff
@@ -215,6 +215,15 @@ _usnic_files = \
 
 if HAVE_VERBS
 _usnic_files += prov/usnic/src/usdf_fake_ibv.c
+
+# If the verbs or usnic providers are being built as a DL, then we
+# need to link libibverbs into the usnic provider
+if HAVE_VERBS_DL
+_usnic_libibverbs = -libverbs
+endif
+if HAVE_USNIC_DL
+_usnic_libibverbs = -libverbs
+endif
 endif
```

Comment:
@goodell @pmmccorm Actually, wait: I can't do this.  I.e., I can't shift the logic to configure.m4 because it's not known / guaranteed that the verbs provider will be configured before the usnic provider.  Hence, the usnic configure.m4 may not know if the verbs provider is going to be built or not.  

As such, the logic has to stay here in Makefile.am.